layouts/partials/head.html

```diff
@@ -119,6 +119,10 @@
         </script>
     {{ end }}
 
+    <!-- Preload the component JavaScript to help our Google Page Speed score. -->
+    <link rel="preload" href="/js/components/components.esm.js" as="script">
```

Comment:
Can you share some data on this? The smaller bundle (which is the one that should be loaded nearly all of the time) is like < 10K over the wire.

Reply:
Page Speed Score: https://developers.google.com/speed/pagespeed/insights/?url=https%3A%2F%2Fpulumi.com%2F

I'm following the instructions they output. I'm playing around with this to see what we get back by preloading. My plan was if I end up going forward with this to add more details to the description.

<img width="947" alt="pagespeedpreload" src="https://user-images.githubusercontent.com/15146337/89963230-e4caed80-dbfb-11ea-83a5-43ca5b713af8.png">